src/components/organisms/browse/List.jsx

```diff
@@ -57,7 +71,10 @@ LastUpdate.propTypes = {
   date: PropTypes.object.isRequired,
 };
 
-export default function FridgeList({ fridges }) {
+export default function FridgeList({ fridges, setCurrentMarker }) {
```

Comment:
It is generally not considered a very good practice to pass the set functions return by useState as props. 
This couples components to closely together and usually indicates a few larger issues:
1) improper component design
2) improper state management.

Reply:
should I use context for it ?